docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -93,6 +95,10 @@ The restore take a few seconds, after which expect to be logged out.
 
```

Comment:
under Configure standby instances above, Create is never clicked?

Reply:
Added "click create"